recipes/configure-capture-agent-manager-gunicorn.rb

```diff
@@ -0,0 +1,25 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-capture-agent-manager-gunicorn
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+app_name = get_capture_agent_manager_app_name
+usr_name = get_capture_agent_manager_usr_name
+
+execute "install gunicorn" do
+  command "source /home/#{usr_name}/sites/#{app_name}/venv/bin/activate" \
```

Comment:
yes, it does. how much re-usable a recipe should be? i could write this assuming that the app requirements already run and just install the gunicorn-start script?